src/App.scss

```diff
@@ -1,2 +1,3 @@
-// Importing Global Variables
 @import "~@redhat-cloud-services/frontend-components-utilities/styles/_all";
+@import '~@patternfly/patternfly/sass-utilities/_all';
+@import 'node_modules/@patternfly/react-core/dist/styles/base.css';
```

Comment:
Do we need these two style files? Starter app doesn't require them so I'd rather keep it simple.